src/Misc/layoutroot/run-helper.sh.template

```diff
@@ -35,7 +35,7 @@ elif [[ $returnCode == 2 ]]; then
 elif [[ $returnCode == 3 ]]; then
     # Wait for 30 seconds or for flag file to exists for the runner update process finish
     echo "Runner listener exit because of updating, re-launch runner after successful update"
-    for i in {0..30} do
+    for i in {0..30}; do
```

Comment:
^ this change was only made in `_layout/run-helper.sh`, not in the source so it wasn't checked in